Learn the Basics/main.py

```diff
@@ -85,10 +85,27 @@ def print_student_info(student):
 
 
 def print_all_records():
+    student_count = 0
     for student in students_list:
         print_student_info(student)
+        student_count = student_count + 1
+    print("The number of students on record is : ",student_count)
```

Comment:
This can work but best practise is fstrings like `print(f"Blah Blah {variable} Blah")`